src/format/context/input.rs

```diff
@@ -117,6 +119,26 @@ impl Input {
         }
     }
 
+    #[inline]
+    pub fn seek_frame(
+        &mut self,
+        stream_index: usize,
+        timestamp: i64,
+        flags: SeekFlags,
+    ) -> Result<(), Error> {
+        unsafe {
+            match av_seek_frame(
+                self.as_mut_ptr(),
+                stream_index as c_int,
```

Comment:
I suppose this can panic if stream_index is high enough and compiled on a 64bit machine. I suggest you change stream_index to be a c_int, or i32 (which c_int is almost always), for safety.

Reply:
Yea this problem is wide-spread in the code-base. For now I prioritize keeping the lib API consistent over panic safety. This issue tracks the problem -> https://github.com/flavioroth/ffmpeg-rs/issues/21